etc/config/c.defaults.properties

```diff
@@ -1,11 +1,48 @@
 # Default settings for C
-compilers=/usr/bin/gcc-4.4:/usr/bin/gcc-4.5:/usr/bin/gcc-4.6:/usr/bin/gcc-4.7:/usr/bin/gcc-4.8:/usr/bin/clangcc:/usr/bin/gcc-5:/usr/bin/gcc-7:/usr/bin/gcc-6:/usr/bin/gcc
-defaultCompiler=/usr/bin/gcc
-postProcess=
+compilers=:&gcc:&clang
+defaultCompiler=gdefault
 demangler=c++filt
 objdumper=objdump
+postProcess=
 supportsBinary=true
 binaryHideFuncRe=^(__.*|_(init|start|fini)|(de)?register_tm_clones|call_gmon_start|frame_dummy|\.plt.*)$
 stubRe=\bmain\b
 stubText=int main(void){return 0;/*stub provided by Compiler Explorer*/}
 supportsLibraryCodeFilter=true
+
+group.gcc.compilers=g44:g45:g46:g47:g48:g5:g6:g7:g8:gdefault
+compiler.g44.exe=/usr/bin/gcc-4.4
```

Comment:
Compilers need to have unique names (for the API). We usually put a c in front of the name here, so cg44 in this case.

Reply:
Updated!